editor/src/app/sites/site.component.ts

```diff
@@ -34,15 +37,15 @@ import { DeleteSiteAction, CloneSiteAction, UpdateSiteAction, RenameSiteAction }
         </div>
         <div class="url-line">
           <a [routerLink]="['/multisite']"
-             [queryParams]="(site.name === '' ? null : {site: site.name})">{{ hostname }}/</a>
+              [queryParams]="(site.name === '' ? null : {site: site.name})" class="link">{{ hostname }}/</a>
 
-          <berta-inline-text-input *ngIf="!modificationDisabled"
-                                   [value]="site.name"
-                                   (inputFocus)="updateComponentFocus($event)"
-                                   (update)="updateField('name', $event)"></berta-inline-text-input>
+          <berta-inline-text-input class="link" *ngIf="!modificationDisabled"
+                                    [value]="site.name"
+                                    (inputFocus)="updateComponentFocus($event)"
+                                    (update)="updateField('name', $event)"></berta-inline-text-input>
         </div>
       </h3>
-    </div>
+    </div>    
```

Comment:
There shouldn't be whitespace at the end of the line. You can set your editor to remove trailing whitespace automatically: https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut